app/Routes.php

```diff
@@ -0,0 +1,147 @@
+<?php
```

Comment:
yes, in different directories.


Muhammad Atif
TL / Sr. Software Engineer @ Coeus Solutions GmbH

muhammad.atif@coeus-solutions.de / +92 (0) 334 7091389 / coeus_m.atif
 (Skype)
Berlin | Munich | Lahore

For more information visit http://www.coeus-solutions.de

Coeus Solutions GmbH // HRB 172 737 // CEO Ali N. Shaheen

Diese E-Mail enthält vertrauliche und/oder rechtlich geschützte
Informationen. Wenn Sie nicht der richtige Adressat sind oder diese E-Mail
irrtümlich erhalten haben, informieren Sie bitte sofort den Absender und
vernichten Sie diese E-Mail.
Das unerlaubte Kopieren sowie die unbefugte Weitergabe dieser E-Mail ist
nicht gestattet.

This e-mail may contain confidential and/or privileged information. If you
are not the intended recipient (or have received this e-mail in error)
please notify the sender immediately and destroy this e-mail.
Any unauthorized copying, disclosure or distribution of the material in
this e-mail is strictly forbidden.

On Wed, Jul 20, 2016 at 11:10 AM, yaseensheikh777 <notifications@github.com>
wrote:

> In app/Routes.php
> <https://github.com/yaseensheikh777/review/pull/1#discussion_r71470387>:
>
> > @@ -0,0 +1,147 @@
> > +<?php
>
> this file is also named as "Routes.php" . Can I make two files with same
> name?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/yaseensheikh777/review/pull/1/files/c6dcb6a777b9cd53a9172a5e62f8e6f09e6f0645#r71470387>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB5DpDaCgfUgB60AhLk0S13P5zoOn7l_ks5qXbvSgaJpZM4JPnuU>
> .
>